compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -4069,15 +4069,23 @@ static const TR::InstOpCode::Mnemonic BinaryArithmeticOpCodesForReg[TR::NumOMRTy
    { TR::InstOpCode::bad, TR::InstOpCode::ADDSSRegReg, TR::InstOpCode::SUBSSRegReg, TR::InstOpCode::MULSSRegReg,  TR::InstOpCode::DIVSSRegReg, TR::InstOpCode::bad,  TR::InstOpCode::bad, TR::InstOpCode::bad  }, // Float
```

Comment:
These tables only have entries for float & double, perhaps this can be simplified.

Reply:
Sure, but is it ok if we open another PR? I would like to minimize changes in this PR and make sure nothing is functionally broken. The main danger is that if I missed some table or some other place that needs to be changed...

Reply:
Sure, I suppose that can wait